api/caseshare/stub-api.ts

```diff
@@ -0,0 +1,95 @@
+import { plainToClass } from 'class-transformer';
+import { Response } from 'express';
+import { SharedCase } from '../../src/cases/models/case-share/case-share.module';
```

Comment:
Please don't mix models between api layer and angular. We've had issues with this in the past